tests/sdk/queries/fileevents/filters/test_activity_filter.py

```diff
@@ -0,0 +1,28 @@
+from tests.sdk.queries.conftest import IS
+
+from py42.sdk.queries.fileevents.filters.activity_filter import RemoteActivity
+from py42.sdk.queries.fileevents.filters.activity_filter import TrustedActivity
+
+
+def test_risk_indicator_mime_type_is_true_str_gives_correct_json_representation():
+    _filter = RemoteActivity.is_true()
+    expected = IS.format("remoteActivity", "TRUE")
```

Comment:
I noticed something strange when playing with boolean filters in the UI --

These filters will send the string `"TRUE" or "FALSE"`:
- remoteActivity
- shared
These filters will send the boolean value `true` or `false`:
- trusted
- mimeTypeMismatch
- outsideActiveHours

It looks like they are currently all configured to work the first way. Have you tested these / do they all work this way?

Reply:
No I didn't test all of them again, I just followed review comments :(

Reply:
I'll check on these to see if anything needs to be changed here, it's possible that something on the backend is converting these nicely and that everything is fine, but we might have to handle the two different cases if not :-/.

Reply:
**Just checked with string values for Trusted activity and found results.**
```

>>> query = FileEventQuery.all(activity_filter.TrustedActivity.is_false())
>>> file_events = sdk.securitydata.search_file_events(query)
```
```

POST    https://forensicsearch-east.us.code42.com/forensic-search/queryservice/api/v1/fileevent
  data "{\"groupClause\":\"AND\", \"groups\":[{\"filterClause\":\"AND\", \"filters\":[{\"operator\":\"IS\", \"term\":\"trusted\", \"value\":\"FALSE\"}]}], \"pgNum\":1, \"pgSize\":10000, \"srtDir\":\"asc\", \"srtKey\":\"eventId\"}"
```

```
>>> file_events.data
{'totalCount': 67364, 'fileEvents': [{'eventId': '0_1d71796f-af5b-4231-9d8e-df6434da4663_952015626463971705_952016152312252793_35', 'eventType': 'CREATED', 'eventTimestamp': '2020-04-29T20:50:03.240Z', 'insertionTimestamp': '2020-04-29T20:50:48.885Z', 'fieldErrors': [], 'filePath': ......
```

Reply:
Found `MimetypeMismatch` and `OutsideActiveHours` returning results with below queries.
```
>>> query = FileEventQuery.all(event_filter.MimeTypeMismatch.is_false())
>>> query = FileEventQuery.all(event_filter.OutsideActiveHours.is_false())
```